docs/guide/previews.md

```diff
@@ -0,0 +1,161 @@
+---
+layout: default
+title: Previews
+parent: Building ViewComponents
+---
+
+# Previews
+
+`ViewComponent::Preview`, like `ActionMailer::Preview`, provides a quick way to preview components in isolation.
+
+_For a more interactive experience, consider using [ViewComponent::Storybook](https://github.com/jonspalmer/view_component_storybook)._
+
+`ViewComponent::Preview`s are defined as:
```

Comment:
```suggestion
`ViewComponent::Previews` are defined as:
```

It was looking weird
![image](https://user-images.githubusercontent.com/11280312/113427695-23840280-939b-11eb-9e03-4247296ee823.png)